src/ontodia/customization/props.ts

```diff
@@ -40,7 +40,8 @@ export interface LinkStyle {
     };
     markerSource?: LinkMarkerStyle;
     markerTarget?: LinkMarkerStyle;
-    labels?: LinkLabelStyle[];
+    label?: LinkLabelStyle;
+    properties?: LinkLabelStyle[];
```

Comment:
What's the meaning of the `properties` array? Does it's order has a specific meaning, like style for the first, the second, etc properties?